src/components/Card/Card.tsx

```diff
@@ -0,0 +1,47 @@
+
+import { SIZE_TYPE, Source } from '../../types';
+import { ButtonProps } from '../Button/Button';
+import { BodyCard, CardBtn, CardStyled, ImageCard, Description, Title, TagsContainer, Tag, Row, SourceCard, DateCard } from './style';
+
+export interface CardProps {
+  date: string;
+  image: string;
+  title: string;
+  source: Source;
+  description: string;
+  tags: string[];
+  button: ButtonProps;
+}
+
+const Card = (props: CardProps) => {
+    const convertDate = () => {
```

Comment:
if you think you may use this function again, it may be better to locate it in utils.ts file